### DISCUSSION TITLE
Sometimes I can get 500 error when run project.

### DISCUSSION THREAD

**Discussion by tobistudio** [vercel/next.js#58454] [App Router]
In my local environment, when I development, I can get 500 error suddenly.
so I 'm running again when get this error too.
I think this problem will be get in vecel environment.
In vercel, I would like to get some suggestions on how to fix it.
![image](https://github.com/vercel/next.js/assets/124074472/dea44505-0203-4bdb-8cfe-830190192cfc)
Is this router problem or ...?

**Comment by prkagrawal** [comment]
Can you provide an reproducible example
